src/pages/index.js

```diff
@@ -6,6 +6,13 @@ import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import Popup from "../components/Popup.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authToken: "1634fdcb-9403-4d55-a456-722f52658f70",
+});
 
 export const initialCards = [
```

Comment:
From the 8th sprint all constants (`initialCards` and `settings`) should be in file `constants.js` in folder `utils` to clean up the main file.

Reply:
completed